ansible/roles/contiv/tasks/restart_netmaster.yml

```diff
@@ -0,0 +1,35 @@
+- name: stop netmaster
```

Comment:
Yes, this is needed because sometimes we want to explicitly restart the networking.